OpenMLDB/src/src/main/java/com/_4paradigm/benchmark/OpenMLDBPerfBenchmark_Cluster.java

```diff
@@ -132,12 +136,18 @@ public OpenMLDBPerfBenchmark_Cluster() {
     }
 
     public void create() {
-        System.out.println("CREATE DATABASE IF NOT EXISTS " + database + ";");
-        Util.executeSQL("CREATE DATABASE IF NOT EXISTS " + database + ";", executor);
-        System.out.println("USE " + database + ";");
-        Util.executeSQL("USE " + database + ";", executor);
+        try {
+            System.out.println("CREATE DATABASE IF NOT EXISTS " + database + ";");
+            Util.executeSQL("CREATE DATABASE IF NOT EXISTS " + database + ";", executor);
+            System.out.println("USE " + database + ";");
+            Util.executeSQL("USE " + database + ";", executor);
+        }
+        catch (Exception e) {
+            e.printStackTrace();
+            System.exit(1);
```

Comment:
`throw new RuntimeExeception("...", e)` is better, you can add more info in the first arg message, and we can know the whole stack.